src/main/redux/sagas/reader.ts

```diff
@@ -147,6 +139,11 @@ async function openReader(publication: Publication, manifestUrl: string) {
         readerWindow.setMenu(null);
     }
 
+    /**
+     * watcher to record windows rectangle position in the db
+     */
+    initRectangleWatcher(readerWindow);
```

Comment:
Naming suggestion: `initWindowRectangleWatcher` ("rectangle" on its own is vague / generic)

Reply:
well seen, more clear, thanks